app/renderer/js/main.ts

```diff
@@ -4,22 +4,22 @@ import { ipcRenderer, remote, clipboard, shell } from 'electron';
 import isDev from 'electron-is-dev';
 import escape from 'escape-html';
 import electronConnect from 'electron-connect';
-import feedbackHolder from './js/feedback';
+import { feedbackHolder } from './feedback';
 
 const { session, app, Menu, dialog } = remote;
 
 // eslint-disable-next-line import/no-unassigned-import
-require('./js/tray');
-
-import DomainUtil = require('./js/utils/domain-util');
```

Comment:
Why this used to work earlier? Was there a relative addressing at place here, this process used to run from index.html, that's why ?

Reply:
Because we were using `<script src="js/main.js">` and so the current way of importing didn't work out well for typescript so I change it to `<script>require('./js/main.js')</script>` which makes both world happy :)